pkg/collector/corechecks/snmp/report/report_utils.go

```diff
@@ -31,6 +32,18 @@ func getColumnValueFromSymbol(values *valuestore.ResultValueStore, symbol checkc
 	return newValues, nil
 }
 
+func getColumnValueFromSymbolOrEmpty(oid string, values *valuestore.ResultValueStore) map[string]valuestore.ResultValue {
+	if oid == "" {
+		return nil
+	}
+	columnValues, err := getColumnValueFromSymbol(values, checkconfig.SymbolConfig{OID: oid})
+	if err != nil {
+		log.Debugf("error getting index values: %v", err)
```

Comment:
If this method is intended to be generic, it probably shouldn't mention `index` here ?